humanoid_league_team_communication/scripts/TeamCommTest.py

```diff
@@ -1,30 +1,48 @@
 #!/usr/bin/env python3
 
+"""
+This script publishes dummy values for ball, goalpost, position and obstacles for testing the team communication.
+"""
+
 import rospy
-from geometry_msgs.msg import PoseWithCovarianceStamped
-from humanoid_league_msgs.msg import ObstacleRelativeArray, ObstacleRelative
+from geometry_msgs.msg import PoseWithCovarianceStamped, PoseWithCovariance
+from humanoid_league_msgs.msg import ObstacleRelativeArray, ObstacleRelative, PoseWithCertaintyArray, PoseWithCertainty
 
 
 if __name__ == '__main__':
     rospy.init_node("TeamCommTest")
-    ball_pub = rospy.Publisher("ball_relative", PoseWithCovarianceStamped, queue_size=1)
-    goal_post_pub = rospy.Publisher("goal_post_relative", PoseWithCovarianceStamped, queue_size=1)
+    ball_pub = rospy.Publisher("balls_relative", PoseWithCertaintyArray, queue_size=1)
+    goal_post_pub = rospy.Publisher("goal_post_relative", PoseWithCertaintyArray, queue_size=1)
     position_pub = rospy.Publisher("position", PoseWithCovarianceStamped, queue_size=1)
```

Comment:
The robot pose is currently published on the topic `pose_with_certainty` and it is also a `PoseWithCertainty` Msg.